### DISCUSSION TITLE
ALARM 1020  with released binaries

### DISCUSSION THREAD

**Discussion by i435f34** [Yaskawa-Global/motoros2#281] [Troubleshooting] [answered]
Hello,

I'm trying to  install MotoROS2 by following the instruction in the README but I've been stuck for a while. After loading the binary on the controller, I always get the following error:

`ALARM 1020 MOTOPLUS APPLICATION LOAD ERROR [5]`

I've checked the troubleshooting for this error and I don't really understand why there would be missing symbols. I'm not building the binaries myself, only uploading the `mr2_yrc1m_h,out`  I've downloaded here https://github.com/yaskawa-global/motoros2/releases. 

I've also tried uploading older versions of MotoROS2 and checking the integrity of the .out file but always get that error.

Setup:
- Yaskawa MotoMini 
- YRC1000micro controller YBS2.31.00-00

Thank you in advance and let me know if you need more details

**Comment by gavanderhoorn** [comment]
I realise we have `YBS2.31.00-00` listed as the oldest version of system sw on YRC1 micro supported, but there might be a chance that should be updated to a newer version in our documentation.

@ted-miller: could you perhaps check?

**Reply by ted-miller** [reply]
I just tested YBS2.31 and it's definitely not compatible. 

@i435f34 You can contact me directly using the email address on my [profile page](https://github.com/ted-miller). I will send you the latest YRC1000micro software. Please note that I only have access to software for English and Japanese languages. If your controller is configured for another language, you'll need to reach out to your [local Yaskawa office](https://www.motoman.com/en-us/about/company/locations) for a localized version.

@gavanderhoorn The micro is a bit harder to work with. But I'll try to determine today what the minimum version is today.

**Reply by i435f34** [reply]
Thank you very much, I've sent you an email @ted-miller

**Reply by gavanderhoorn** [reply]
I'd like to keep this *unanswered* until we have confirmation the problem(s) you encounter @i435f34 are solved.

**Reply by ted-miller** [reply]
The missing function is `mpCtrlGrpNo2GrpId`. But none of my release notes indicate when that function was added. I'll need to reach out to the Yaskawa Japan office for clarification.

**Comment by gavanderhoorn** [comment]
Tracking in #285.

**Reply by gavanderhoorn** [reply, marked as answer]
Turns out `2.45` is the minimum version. Updated in #286. Thanks @ted-miller 👍

**Reply by gavanderhoorn** [reply]
And thank you @i435f34 for reporting.
